addons/ofxAssimpModelLoader/src/ofxAssimpModelLoader.cpp

```diff
@@ -253,7 +253,7 @@ void ofxAssimpModelLoader::loadGLResources(){
             
             ofxAssimpTexture assimpTexture;
             bool bTextureAlreadyExists = false;
-            for(int j=0; j<textures.size(); j++) {
+            for(unsigned int j=0; j<textures.size(); j++) {
```

Comment:
Using `size_t` consistently for all comparison against `.size()`  would be nice... I know on most systems `unsigned int` is probably the same type as `decltype(textures.size())`, but it might not always be.